examples/src/main/python/ml/kmeans_example.py

```diff
@@ -19,9 +19,10 @@
 
 # $example on$
 from pyspark.ml.clustering import KMeans
+from pyspark.sql import SparkSession
```

Comment:
Similar comment as previous reordering of SparkSession import, consider moving $example off$ above this import so examples remain the same.